utilities/copyoffload_plan_secret.py

```
@@ -0,0 +1,45 @@
+    try:
+        for sample in TimeoutSampler(
+            wait_timeout=60,
+            sleep=2,
+            func=lambda: any(
+                s.name.startswith(f"{plan_name}-") for s in Secret.get(client=ocp_admin_client, namespace=namespace)
+            ),
+        ):
+            if sample:
+                return
+    except TimeoutExpiredError:
+        LOGGER.warning(f"Timeout waiting for plan secret '{plan_name}-*' - continuing anyway")
```

Ruff (0.15.15)

[warning] Logging statement uses f-string

(G004)